errors.go

```diff
@@ -13,21 +13,26 @@ import (
 const (
 	// formatErrUnknown is used to create an dynamic error if no error matches
 	formatErrUnknown = "Unknown Error (%d): %s"
+	jsonParsingError = "error decoding json response (%s): %s"
+
 	// fieldReqMsg is API error stating a field is required.
-	fieldReqMsg       = "This field is required."
-	invalidUserMsg    = "Enter a valid username. This value may contain only letters, numbers and @/./+/-/_ characters."
-	failedLoginMsg    = "Unable to log in with provided credentials."
-	invalidAppNameMsg = "App name can only contain a-z (lowercase), 0-9 and hyphens"
-	invalidNameMsg    = "Can only contain a-z (lowercase), 0-9 and hyphens"
-	invalidCertMsg    = "Could not load certificate"
-	invalidPodMsg     = "does not exist in application"
-	invalidDomainMsg  = "Hostname does not look valid."
-	invalidVersionMsg = "version cannot be below 0"
-	invalidKeyMsg     = "Key contains invalid base64 chars"
-	duplicateUserMsg  = "A user with that username already exists."
-	invalidEmailMsg   = "Enter a valid email address."
-	invalidTagMsg     = "No nodes matched the provided labels"
-	duplicateIDMsg    = "App with this id already exists."
+	fieldReqMsg           = "This field is required."
+	invalidUserMsg        = "Enter a valid username. This value may contain only letters, numbers and @/./+/-/_ characters."
+	failedLoginMsg        = "Unable to log in with provided credentials."
+	invalidAppNameMsg     = "App name can only contain a-z (lowercase), 0-9 and hyphens"
+	invalidNameMsg        = "Can only contain a-z (lowercase), 0-9 and hyphens"
+	invalidCertMsg        = "Could not load certificate"
+	invalidPodMsg         = "does not exist in application"
+	invalidDomainMsg      = "Hostname does not look valid."
+	invalidVersionMsg     = "version cannot be below 0"
+	invalidKeyMsg         = "Key contains invalid base64 chars"
+	duplicateUserMsg      = "A user with that username already exists."
+	invalidEmailMsg       = "Enter a valid email address."
+	invalidTagMsg         = "No nodes matched the provided labels"
+	duplicateIDMsg        = "App with this id already exists."
+	cancellationFailedMsg = "still has applications assigned. Delete or transfer ownership"
+	duplicateDomainMsg    = "Domain is already in use by another application"
+	duplicateKeyMsg       = "Public Key is already in use"
```

Comment:
I'm guessing this really only has a line or two changed. because I'm lazy, can you tell me what actually changed here?